docker-bake.hcl

```
@@ -154,6 +155,21 @@ target "debian_slim_jdk8" {
   platforms = ["linux/amd64", "linux/ppc64le", "linux/arm64"]
 }
 
+target "rhel_ubi8_jdk11" {
+  dockerfile = "11/rhel/ubi8/hotspot/Dockerfile"
+  context = "."
+  args = {
+    JENKINS_VERSION = JENKINS_VERSION
+    JENKINS_SHA = JENKINS_SHA
+  }
+  tags = [
+    "${REGISTRY}/${JENKINS_REPO}:${JENKINS_VERSION}-rhel-ubi8-jdk11",
+    equal(LATEST_WEEKLY, "true") ? "${REGISTRY}/${JENKINS_REPO}:rhel-ubi8-jdk11" : "",
+    equal(LATEST_LTS, "true") ? "${REGISTRY}/${JENKINS_REPO}:lts-rhel-ubi8-jdk11" : "",
+  ]
+  platforms = ["linux/amd64"]
```

is there a reason we shouldn't ship this on all the supported platforms?
```suggestion
  platforms = ["linux/amd64", "linux/arm64", "linux/ppc64ls", "linux/s390x"]
```

---

None that I know of. Just opted to make the image set smaller than larger when Ubi inclusion is being considered. I do not mind adding all the platforms.